src/Compilers/CSharp/Test/Symbol/Symbols/AnonymousTypesSemanticsTests.cs

```diff
@@ -767,8 +767,8 @@ void m()
             Instance Receiver: 
               null
         Right: 
-          IOperation:  (OperationKind.None, Type: null) (Syntax: 'x')
-      IOperation:  (OperationKind.None, Type: null) (Syntax: 'x')
+          IParameterReferenceOperation: x (OperationKind.ParameterReference, Type: System.Int32) (Syntax: 'x')
+      IParameterReferenceOperation: x (OperationKind.ParameterReference, Type: System.Int32) (Syntax: 'x')
```

Comment:
>IParameterReferenceOperation: x (OperationKind.ParameterReference, Type: System.Int32) (Syntax: 'x') [](start = 6, length = 100)

It doesn't look like the initializers are in the right order.

Reply:
Tracked with https://github.com/dotnet/roslyn/issues/23115